src/Dropdown/__snapshots__/Dropdown.spec.js.snap

```diff
@@ -42,16 +42,24 @@ exports[`Dropdown 1`] = `
 }
 
 <div
-  aria-expanded={false}
-  aria-haspopup="true"
-  className="re-dropdown-trigger c0"
+  onBlur={[Function]}
+  onMouseLeave={[Function]}
+  tabIndex={0}
 >
-  <button
-    className="re-button re-button-primary c1"
-    disabled={false}
-    size="md"
+  <div
+    aria-expanded={false}
+    aria-haspopup="true"
+    className="re-dropdown-trigger c0"
+    onClick={[Function]}
+    onMouseEnter={[Function]}
   >
-    Trigger
-  </button>
```

Comment:
It seems like more tests would be a good idea here? I will add some more.

Reply:
can we add snapshots after the dropdown has been opened/closed to test event propagation?

Reply:
This seems possible: https://jestjs.io/docs/en/tutorial-react#snapshot-testing

Reply:
It turns out we can use our existing dependencies for this. [Another branch](url) has been set up for that work (r[elevant commit](https://github.com/sappira-inc/molekule/commit/80697576237ec957efc0d27f302bf2b1fd927316)), so that it doesn't block this.

Reply:
do you want to roll it into this PR or is this otherwise ready to go?

Reply:
I've set up a [PR](https://github.com/sappira-inc/molekule/pull/25) to roll it into this PR if that's alright?